src/Symfony/Component/Messenger/Bridge/Amqp/Compressor/CompressorFactory.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Bridge\Amqp\Compressor;
+
+use Symfony\Component\Messenger\Exception\InvalidArgumentException;
+
+class CompressorFactory
+{
+    public static function createCompressor(string $mimeContentEncoding)
+    {
+        if ('gzip' === $mimeContentEncoding) {
```

Comment:
May be move to const into the class Gzip?

Reply:
Hi! You can check if this is what you had in mind. Thanks.